### PR TITLE
fix: high-band STFT loss slices time frames instead of frequency bins

### PR DESCRIPTION
**Pull request by andrewwhitecdw**
### Problem
fix: high-band STFT loss slices time frames instead of frequency bins

### Fix
Replace:
```
        if self.band == "high":
            freq_mask_ind = x_mag.shape[1] // 2  # only select high frequency bands
            sc_loss  = self.spectral_convergence_loss(x_mag[:,freq_mask_ind:,:], y_mag[:,freq_mask_ind:,:])
            mag_loss = self.log_stft_magnitude_loss(x_mag[:,freq_mask_ind:,:], y_mag[:,freq_mask_ind:,:])
```

with:
```
        if self.band == "high":
            freq_mask_ind = x_mag.shape[2] // 2  # only select high frequency bands
            sc_loss  = self.spectral_convergence_loss(x_mag[:,:,freq_mask_ind:], y_mag[:,:,freq_mask_ind:])
            mag_loss = self.log_stft_magnitude_loss(x_mag[:,:,freq_mask_ind:], y_mag[:,:,freq_mask_ind:])
```

### Files changed
- `stft_loss.py`

### CODE REVIEW

**Comment by andrewwhitecdw** [comment]
Closing this sweep-generated PR: sole commit is missing a valid Signed-off-by trailer. It does not meet the sweep requirements (single signed-off commit).

**Comment by andrewwhitecdw** [comment]
Closing after the adversarial audit found the proposed change was not safe or complete enough for a small focused PR. A revised fix can be submitted separately. Thanks for understanding.

**Comment by andrewwhitecdw** [comment]
Reopening — the previous bulk-close was issued in error. No action needed from maintainers.
